### PR TITLE
Updates for 3.0

### LINKED ISSUE #109: Package needs to be updated to point to @dojo/framework
This package should be updated to use the new `@dojo/framework` instead of the individual core, has, i18n, widget-core, routing, stores, and shim packages.

### PR DESCRIPTION
**Pull request by bryanforbes**
Resolves #109

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +33,14 @@
...
     ]
   },
+  "peerDependencies": {
+    "@dojo/framework": "file:./tarballs/dojo-framework-3.0.0-pre.tgz",
+    "@dojo/webpack-contrib": "file:./tarballs/dojo-webpack-contrib-3.0.0-alpha.1.tgz"
+  },
   "devDependencies": {
-    "@dojo/cli": "2.0.0",
+    "@dojo/cli": "file:./tarballs/dojo-cli-3.0.0-alpha.1.tgz",
+    "@dojo/shim": "^2.0.0",
```
Comment (agubler) [inline package.json]: Still needed?

Comment (bryanforbes) [reply package.json]: Evidently, no. At one point yesterday, I was adding things until it worked and this was one of those things.

File: package.json [hunk 2]
```diff
@@ -17,14 +17,25 @@
...
+    "build:static:dev": "cpx \"{src,tests}/**/*.{d.ts,html,md,json,js,css}\" dist/dev",
+    "build:static:release": "cpx \"src/**/*.{d.ts,json}\" dist/release",
+    "build:cjs": "tsc",
+    "build": "concurrently \"npm:build:*\" && dojo-package",
+    "clean": "rimraf dist coverage test-app/output",
+    "lint": "tslint -p .",
+    "precommit": "lint-staged",
+    "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
     "test:ci": "run-p --race serve test",
+    "test": "npm run build && npm run artifact:package && npm run artifact:install && npm run build && intern && npm run artifact:build && cypress run",
```
Comment (devpaul) [inline package.json]: We should add `prettier -l {src,tests}/**/*.ts` to fail the build if files don't comply

File: src/dev.config.ts [hunk 3]
```diff
@@ -32,7 +32,9 @@ function webpackConfig(args: any): webpack.Configuration {
 			new WebpackPwaManifest({
 				...manifest,
 				ios: true,
-				icons: Array.isArray(manifest.icons) ? manifest.icons.map(icon => ({ ...icon, ios: true })) : manifest.icons
+				icons: Array.isArray(manifest.icons)
```
[the remark this answers is not in the document]
Comment (bryanforbes) [reply src/dev.config.ts]: The prettier config wasn't in `package.json`. I'll add it.

**Pull request merged**